app/assets/stylesheets/components/_alert.scss

```diff
@@ -1,43 +1,20 @@
 .alert {
   background-color: $blue-light;
-  border-radius: $border-radius;
-  color: $gray;
-  margin-bottom: 2.5rem;
-  padding: 1.75rem;
-  position: relative;
-
-  &::before {
-    left: 0;
-    margin-left: auto;
-    margin-right: auto;
-    position: absolute;
-    right: 0;
-    top: -16px;
-    width: 32px;
-  }
+  color: $black;
 }
 
 .alert-notice,
 .alert-success {
   background-color: #f0faed;
-
-  &::before {
-    content: url(image-path('alert/ico-check.svg'));
-  }
+  color: #088a22;
```

Comment:
i think we should name them if we end of using them in at least one other place; i prefer not to create variables for single usage css